content/reference/cd-supporting-new-ci.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Supporting A New CI System
+aliases: ["cd-supporting-new-ci.html"]
```

Comment:
This line is not needed and should be deleted. Existing files that had previously been published with Jekyll using the ugly URLs have these aliases (redirects), but new content doesn’t need it.

```suggestion
```

Reply:
Done.